tests/test_coreg.py

```diff
@@ -126,3 +131,176 @@ def test_only_paths():
     to_be_aligned_raster = examples.FILEPATHS["longyearbyen_tba_dem"]
 
     coreg.coregister(reference_raster, to_be_aligned_raster)
+
+
+class TestCoregClass:
+    ref, tba, outlines = load_examples()  # Load example reference, to-be-aligned and mask.
+    mask = outlines.create_mask(ref) == 255
+
+    fit_params = dict(
+        reference_dem=ref.data,
+        dem_to_be_aligned=tba.data,
+        mask=~mask,
+        transform=ref.transform
+    )
+    # Create some 3D coordinates with Z coordinates being 0 to try the apply_pts functions.
+    points = np.array([[1, 2, 3, 4], [1, 2, 3, 4], [0, 0, 0, 0]], dtype="float64").T
+
+    def test_bias(self):
+        warnings.simplefilter("error")
+
+        # Create a bias correction instance
+        biascorr = coreg.BiasCorr()
+        # Fit the bias model to the data
+        biascorr.fit(**self.fit_params)
+
+        # Check that a bias was found.
+        assert biascorr._meta.get("bias") is not None
+        assert biascorr._meta["bias"] != 0.0
+
+        # Copy the bias to see if it changes in the test (it shouldn't)
+        bias = copy.copy(biascorr._meta["bias"])
+
+        # Check that the to_matrix function works as it should
+        matrix = biascorr.to_matrix()
+        assert matrix[2, 3] == bias, matrix
```

Comment:
How does the `,` works for the `==` assertion?

Reply:
Everything after the comma gets printed in the `AssertionError`:
```python
assert 1 == 2, "One turns out to be different than two"
```
```
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-1-d33f3b548b72> in <module>
----> 1 assert 1 == 2, "One turns out to be different than two"

AssertionError: One turns out to be different than two
```
Just printing the `matrix` variable is not very helpful any more, I confess. It was for me to check it visually when I developed it!